sdk/communication/Azure.Communication.ProgrammableConnectivity/tests.yml

```diff
@@ -0,0 +1,7 @@
+trigger: none
+
+extends:
```

Comment:
This should live under `/sdk/communication` - not this specific library.

Reply:
Out of all of the libraries that live in `sdk/communication`, almost all of them have their own `tests.yml` file. I don't see  a `tests.yml` file in `sdk/communication`. Could you please let me know what I'm getting wrong here?

Reply:
The `tests.yml` file creates the live test pipeline.   You shouldn't need one of these if you're bypassing live tests due to special needs.    Generally, we do not want them under individual packages because it creates a bunch of junk pipelines, but since the ACS team is reviewing/approving/merging ACS PRs, the expected pattern is not being followed.

Reply:
have removed